src/ansys/dpf/core/property_field.py

```diff
@@ -91,7 +91,10 @@ def __init__(
             field=property_field,
             server=server,
         )
-        self._field_definition_instance = None
+        if meets_version(self._server.version, "11.0"):
+            self._field_definition_instance = self._load_field_definition()
```

Comment:
why doing it at __init__ and not lazy anymore? It can have non negligeable impact on performance